### ISSUE TITLE
statement id after execute

### ISSUE THREAD

**Issue by exaSR** [exasol/websocket-api#28]
It would be really cool if the `execute` and `executeBatch` calls would return an array of Exasol statement IDs that match the executed statement(s)...

**Comment by littleK0i** [comment]
Yes, it would be awesome. At this moment I have to manually count statements in pyexasol, and sometimes this counter will not match real STMT_ID.

**Comment by rrusinov** [comment]
Thank you for reaching out and sharing your feature request regarding the enhancement of the `execute` and `executeBatch` calls to return an array of Exasol statement IDs. We understand how such a feature could significantly improve usability and efficiency, especially in scenarios where tracking and managing statement IDs is crucial for your workflow.

However, it's important to note that the scope of this GitHub repository is focused solely on providing documentation for the WebSocket protocols supported by the Exasol database. As such, we don't directly implement or modify the functionalities of the Exasol database, including the protocol definition of database operations like `execute` and `executeBatch`.

That said, we highly value your input and recognise the potential benefits of your suggestion. To ensure your request is directed to the appropriate team who can evaluate and potentially implement this feature, we recommend reaching out to the Exasol database support and product management team. You can contact them through the regular first-level support channels provided to Exasol customers.
